app/src/main/AndroidManifest.xml

```diff
@@ -1,21 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="edu.uc.it3048.mypokedex">
-    <uses-permission android:name="android.permission.INTERNET"/>
+<manifest xmlns:android = "http://schemas.android.com/apk/res/android"
+    xmlns:tools = "http://schemas.android.com/tools"
+    package = "edu.uc.it3048.mypokedex">
+    <uses-permission android:name = "android.permission.INTERNET"/>
     <application
         android:allowBackup="true"
```

Comment:
these couple did not need changed because they were automatically generated by android studio